### ISSUE TITLE
Training models with high noise levels does not work properly

### ISSUE THREAD

**Issue by PTRRupprecht** [HelmchenLabSoftware/Cascade#57]
The ground truth for models with high noise levels becomes smaller and smaller (it should instead become larger).

**Comment by PTRRupprecht** [comment]
The bug was introduced by a modification made a few months ago (Jul 30, 2023). No pretrained models available via Github are affected. The problem was an error thrown when a gradient descent algorithm in the function `calibrated_ground_truth_artificial_noise()` in `casdade2p\utils.py` threw an error. A simple change (decrease of gradient step size) solved the problem.
